src/test/project-converter_test.ts

```diff
@@ -125,20 +146,21 @@ suite('AnalysisConverter', () => {
       setSources({
         'test.html': `
           <link rel="import" href="./dep.html">
-          <link rel="import" href="../dep.html">
+          <link rel="import" href="../dep/dep.html">
           <script></script>
         `,
         'dep.html': `<h1>Hi</h1>`,
-        'bower_components/dep.html': `<h1>Hi</h1>`,
+        'bower_components/dep/dep.html': `<h1>Hi</h1>`,
       });
       const expectedWarnings = [
-        `WARN: bower->npm mapping for "dep.html" not found`,
-        `WARN: bower->npm mapping for "dep.html" not found`,
+        `WARN: bower->npm mapping for "dep" not found`,
```

Comment:
I thought there was some kind of memoization in the lookup to reduce duplicate warnings?  Maybe I misunderstood.  In any case it might be good to state in a comment why 3 duplicate warnings are expected.

Reply:
Haha almost, I was explaining that that would be a nice feature that's easier to implement now :) Still TODO.